unisave-rest/src/main/java/org/uniprot/api/unisave/request/UniSaveRequest.java

```diff
@@ -0,0 +1,34 @@
+package org.uniprot.api.unisave.request;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+
+import lombok.Data;
+
+/**
+ * Created 27/03/20
+ *
+ * @author Edd
+ */
+public class UniSaveRequest {
+    @Data
+    public static class Entries {
+        private String accession;
```

Comment:
not sure if necessary here, but maybe validate the accession input with the famous Regex...

Reply:
done (for all uses of an accession in endpoints)